conda_smithy/templates/run_docker_build.sh.tmpl

```diff
@@ -65,7 +69,9 @@ DOCKER_RUN_ARGS="${CONDA_FORGE_DOCKER_RUN_ARGS}"
 if [ -z "${CI}" ]; then
     DOCKER_RUN_ARGS="-it ${DOCKER_RUN_ARGS}"
 fi
+endgroup "Configure Docker"
 
+startgroup "Start Docker"
```

Comment:
This is missing an end group

Reply:
Ohhh this is matched in the docker script. Clever. A comment to that effect here and in the place where it is matched would be good.

Reply:
The corresponding endgroup is called on `build_steps`.

Reply:
Yup. Adding a comment will help other devs who expect it to be matched in the same file.

Reply:
Oh, sorry, second comment didn't load and I was replying to the first (that happens often, btw, don't know if that's a problem on GitHub end or a browser plugin interfering).

Should I use a Bash comment so it's visible on the rendered output or just a Jinja comment on the template?

Reply:
A bash comment will make it visible in both spots and should be good. Thanks!

Reply:
Jinja comment is better

Reply:
LOL ok. 🤦 

I do not care even a little bit.

Reply:
Done 😄